rules/limit-language-features.js

```diff
@@ -66,30 +68,33 @@ module.exports = {
-    'selector-pseudo-class-blacklist': null, // Specify a blacklist of disallowed pseudo-class selectors.
-    'selector-pseudo-class-whitelist': null, // Specify a whitelist of allowed pseudo-class selectors.
-    'selector-pseudo-element-blacklist': null, // Specify a blacklist of disallowed pseudo-element selectors
-    'selector-pseudo-element-whitelist': null, // Specify a whitelist of allowed pseudo-element selectors
+    'selector-pseudo-class-disallowed-list': null, // Specify a list of disallowed pseudo-class selectors.
+    'selector-pseudo-class-allowed-list': null, // Specify a list of allowed pseudo-class selectors.
+    'selector-pseudo-element-disallowed-list': null, // Specify a list of disallowed pseudo-element selectors.
+    'selector-pseudo-element-allowed-list': null, // Specify a list of allowed pseudo-element selectors.
 
     // Media feature
-    'media-feature-name-blacklist': null, // Specify a blacklist of disallowed media feature names.
+    'media-feature-name-disallowed-list': null, // Specify a list of disallowed media feature names.
     'media-feature-name-no-vendor-prefix': true, // Disallow vendor prefixes for media feature names.
-    'media-feature-name-value-whitelist': null, // Specify a whitelist of allowed media feature name and value pairs.
-    'media-feature-name-whitelist': null, // Specify a whitelist of allowed media feature names.
+    'media-feature-name-value-allowed-list': null, // Specify a list of allowed media feature name and value pairs.
+    'media-feature-name-allowed-list': null, // Specify a list of allowed media feature names.
 
     // Custom media
     'custom-media-pattern': "fs-.+", // Specify a pattern for custom media query names.
 
     // At-rule
-    'at-rule-blacklist': null, // Specify a blacklist of disallowed at-rules.
+    'at-rule-disallowed-list': null, // Specify a list of disallowed at-rules.
     'at-rule-no-vendor-prefix': true, // Disallow vendor prefixes for at-rules.
-    'at-rule-property-requirelist': {}, // Specify a requirelist of properties for an at-rule.
-    'at-rule-whitelist': null, // Specify a whitelist of allowed at-rules.
+    'at-rule-property-required-list': {}, // Specify a list of required properties for an at-rule.
+    'at-rule-allowed-list': null, // Specify a list of allowed at-rules.
 
     // Comment
-    'comment-word-blacklist': ["/^TODO/", { "severity": "warning" }], // Specify a blacklist of disallowed words within comments.
+    'comment-word-disallowed-list': ["/^TODO/", { "severity": "warning" }], // Specify a list of disallowed words within comments.
+    'comment-pattern': null, // Specify a pattern for comments.
 
     // General / Sheet
     'linebreaks': null, // Specify unix or windows linebreaks.
     'max-nesting-depth': [2, {ignore: ['blockless-at-rules', 'pseudo-classes']}], // Limit the depth of nesting.
+    'alpha-value-notation': null, // Specify percentage or number notation for alpha-values.
```

Comment:
Doing a search in Infinity, we only use number, never % for the opacity in `rgba` ?

Reply:
yes you are right! good catch, I guess when I searched throughout our repos my eyes mistakenly took the `%` in cases such as this:
![image](https://user-images.githubusercontent.com/13462129/139969419-1abfa51b-9dc7-407a-a4b9-c12398f95103.png)

for `rgba` :D ...

I think we can set this to `number`; I believe we have used `number` throughout...

Reply:
[little note ](https://github.com/stylelint/stylelint/issues/4718#issuecomment-619377752)that I just came across as I was looking for something else in github. So this guy is author of stylelint (I think lol) and he suggests `percentage` instead of `number` for alpha values. @mingteo-str do you think we like to do this change? That would need a lot of changes across repos (I mean easy changes, and I mean I don't mind doing them since it is all the purpose of this ticket really).

Reply:
currently I have set it to number.